addons/web_editor/static/lib/odoo-editor/src/commands/align.js

```diff
@@ -1,17 +1,16 @@
 /** @odoo-module **/
 import { childNodeIndex, isBlock } from '../utils/utils.js';
 
-Text.prototype.oAlign = function (offset, mode) {
-    this.parentElement.oAlign(childNodeIndex(this), mode);
+Text.prototype.oAlign = function (mode) {
+    this.parentElement.oAlign(mode);
 };
 /**
  * This does not check for command state
- * @param {*} offset
- * @param {*} mode 'left', 'right', 'center' or 'justify'
+ * @param {string} mode 'left', 'right', 'center' or 'justify'
  */
-HTMLElement.prototype.oAlign = function (offset, mode) {
-    if (!isBlock(this)) {
-        return this.parentElement.oAlign(childNodeIndex(this), mode);
+HTMLElement.prototype.oAlign = function (mode) {
+    if (!isBlock(this) && this.nodeName !== 'A') {
```

Comment:
This looks ad-hoc and would be worth at least a comment.

Reply:
True.